surefire-providers/surefire-junit-platform/src/test/java/org/apache/maven/surefire/junitplatform/JUnitPlatformProviderTest.java

```diff
@@ -772,7 +775,8 @@ public void filtersWithEmptyJunitEngineAreNotRegistered()
         JUnitPlatformProvider provider = new JUnitPlatformProvider( providerParameters );
 
         assertThat( provider.getFilters() ).hasSize( 1 );
-        assertThat( provider.getFilters() ).allMatch( x -> x instanceof EngineFilter );
+        assertThat( Arrays.stream( provider.getFilters() ).collect( Collectors.toList()) )
```

Comment:
This is unnecessarily complicated.
I have checked it out in my code, and this works for me:
`assertThat( provider.getFilters() ).asList().element( 0 ).isInstanceOf( EngineFilter.class )`
Does it work for you too?

Reply:
For me assertThat( provider.getFilters() ).asList() throw exception as getFilters() return array but expected was list :/

Reply:
ok, then better do it without the Java Stream like this `assertThat( provider.getFilters()[0] ).isInstanceOf( EngineFilter.class )`